scout/server/extensions/beacon_extension.py

```diff
@@ -219,3 +220,28 @@ def remove_variants(self, store, institute_id, case_obj):
 
         if update_case:
             store.case_collection.update_one({"_id": case_obj["_id"]}, {"$unset": {"beacon": 1}})
+            # Create a relative event in database
+            self.update_event_collection(store, user_obj, case_obj, "beacon_remove")
+
+    def update_event_collection(self, store, user_obj, case_obj, verb):
+        """Update the event database collection whenever variants are added of remove from the BEACON_URL
+
+        Args:
+            store(adapter.MongoAdapter)
+            user_obj(dict): scout.models.
+            case_obj(dict): scout.models.User
+            verb(string): "beacon_add" or "beacon_remove"
+        """
+        institute_obj = store.institute(case_obj["owner"])
+        link = f"/{institute_obj['_id']}/{case_obj['display_name']}"
```

Comment:
`url_for()` - but never mind if it will need a particular context switch when in the extension.

Reply:
Same here